website/docs/d/is_security_group_rule.html.markdown

```diff
@@ -0,0 +1,103 @@
+---
+layout: "ibm"
+page_title: "IBM : security_group_rule"
+sidebar_current: "docs-ibm-resource-is-security-group-rule"
+description: |-
+  Manages IBM Security Group Rule.
+---
+
+# ibm\_is_security_group_rule
+
+Provides a security group rule data source. This allows security group rule to be read.
+
+
+## Example Usage
+
```

Comment:
This is a datasource not resource change the 
In the following example, you can create a different types of protocol rules `ALL`, `ICMP`, `UDP` and `TCP`
In the following example, you can create a different types of protocol rules `ALL`, `ICMP`, `UDP` and `TCP` and read the security group